### DISCUSSION TITLE
Update array of models at once using .merge function

### DISCUSSION THREAD

**Discussion by 3adel-bassiony** [adonisjs/core#3930] [Help] [answered]
Hello
Is there a way to update multiple items with the same type and value using the merge function?
for example something like this, but this will return an error because `merge` not working with arrays

```
        const items = await Item.query().where('name', 'test')
        await items.merge({ deletedAt: DateTime.now() }).save()
```

I know I can use for loop but I'm asking if there is something better to update the array at once

**Comment by tomgobich** [comment, marked as answer]
You can do this all in one call using the `update` method on the query.
```ts
await Item.query().where('name', 'test').update({ deletedAt: DateTime.now() })
```
https://docs.adonisjs.com/guides/models/crud-operations#update

**Reply by 3adel-bassiony** [reply]
@tomgobich Thank you 💪

**Reply by tomgobich** [reply]
Anytime! :)
